### ISSUE TITLE
Can we shorten the podname in grafana using Legend option

### ISSUE THREAD

**Issue by Bharath4297** [bitnami/charts#8321]
In Grafana dashboard, to know how many pods are running we use the query as follows:

```
kube_pod_status_phase{cluster="$cluster",namespace=~"^$namespace",pod=~"^$pod", phase="Running"} > 0
```

and in legend options box we mention as follows:

```
{{phase}}::{{pod}}
```

we are getting the graph and to the left of the graph we are getting info as follows:

```
Running::kube-multus-ds-amd64-zpk9x
Running::kube-proxy-g4cpb
Running::kube-proxy-wv6wb
Running::kube-sriov-cni-ds-amd64-b5klk
Running::kube-sriov-cni-ds-amd64-sv8l9
```

but instead of getting such big name, I just want only pod name i.e

for ex:

- instead of `Running::kube-multus-ds-amd64-zpk9x`, i want `running::kube-multus`
- instead of `Running::kube-proxy-g4cpb`, i want `running::kube-proxy`.

basically I dont't want the whole name i just want the exact name.

Would anyone please help me? thanks in advance.

Note: I am using Prometheus Data Source

**Comment by juan131** [comment]
Hi @Bharath4297 

Please note the actual pod names that belong to certain deployment follow the format `{{DEPLOYMENT_NAME}}-{{RANDOM_SUFFIX}}`. If you're interested on the deployment name, then you need to should use that instead.

In any case, this issue doesn't seem to be related with the Bitnami Grafana chart but how to configure Grafana instead. I highly recommend you to re-post your question at https://github.com/grafana/grafana/issues since that's the appropriate channel to ask questions that are related to Grafana itself.

**Comment by Bharath4297** [comment]
Hi @juan131 ,
Thanks for your support. Sure I will post it in grafana community.

**Comment by juan131** [comment]
Thanks!!
